package.json

```diff
@@ -27,7 +27,7 @@
     "start": "run-p build-uncompressed server watch:**",
     "snyk": "snyk test --file=package.json",
     "test:procfile": "node test/test-procfile.js",
-    "test:eslint": "eslint --config ./.eslintrc.json \"network-api/networkapi/**/*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
+    "test:eslint": "eslint --config ./.eslintrc.json \"network-api/networkapi/{,!(frontend)/**/}*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
```

Comment:
@Pomax is this where you added network api as prettier target?

Reply:
correct, it's a third directory tree that eslint gets to look in.